### DISCUSSION TITLE
Connection refused

### DISCUSSION THREAD

**Discussion by webdantas** [laravel/framework#46246] [Q&A]
- Laravel Version: 9.52.4
- PHP Version: PHP 8.1.12 (cli) (built: Oct 25 2022 17:30:00) (NTS)
- Database Driver & Version:  mysql  Ver 8.0.30-0ubuntu0.20.04.2 for Linux on x86_64 ((Ubuntu))

### Description:
The default configuration of the installation generates a connection refused error, I changed these lines:

### Original
BROADCAST_DRIVER=pusher
CACHE_DRIVER=file
FILESYSTEM_DISK=local
QUEUE_CONNECTION=sync
SESSION_DRIVER=database
SESSION_LIFETIME=120

### Changed
BROADCAST_DRIVER=pusher
CACHE_DRIVER=file
**#** FILESYSTEM_DISK=local
**#** QUEUE_CONNECTION=sync
SESSION_DRIVER=**file** 
SESSION_LIFETIME=120

Now the application connects normally again, which before only the database IDE could connect

**Comment by henzeb** [comment]
and other parts of the application are still able to connect to the database?
